src/components/GradientBackground.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FunctionComponent } from "react"
+import { StyleSheet } from "react-native"
+
+import LinearGradient from "react-native-linear-gradient"
+
+import { Colors } from "../styles"
+
+const GradientBackground: FunctionComponent = () => {
```

Comment:
What do you think about keeping this inline until we have a few places where this is being used?

Reply:
Based on the Figma designs, it looks like we'll be using this in several other spots in the next iteration. Given that, I feel like it makes sense to leave as is.